cookbook/how_to_install_extension_from_command_line.rst

```diff
@@ -82,7 +82,7 @@ Finally, make sure to properly clean the cache:
 Querying Extension Information
 ------------------------------
 
-There are two commands that help you to get an overview about the extension.
+There are two commands that help you get an overview about the extension.
```

Comment:
Pretty sure it is "help + infinitive". Actually "to" is also correct but without to sounds better, imho.